tests/integration/data/schemas/init.ts

```diff
@@ -0,0 +1,58 @@
+import { handler } from '../../../../src/commands/data/schemas/init';
+import { ConsoleSpy, spyOnConsole } from '../../../__helpers__/consoleSpy';
+import { createTempDirectoryManager } from '../../../__helpers__/tempDirectoryManager';
+
+describe('exh data schemas init', () => {
+  let tempDir: Awaited<ReturnType<typeof createTempDirectoryManager>>;
+  let consoleSpy: ConsoleSpy;
+
+  beforeEach(async () => {
+    tempDir = await createTempDirectoryManager();
+    consoleSpy = spyOnConsole();
+  });
+
+  afterEach(async () => {
+    jest.clearAllMocks();
+    await tempDir.removeDirectory();
+  });
+
+  it('Creates a schema', async () => {
+    const path = tempDir.getPath();
+    const name = 'test-schema';
+
+    await handler({ name, path });
```

Comment:
Maybe to discuss: we're leaving a whole lot out of these tests like this, yargs is doing a lot, but is not covered at all